### DISCUSSION TITLE
The IPA (International Phonetic Alphabet) obtained for "has" is inaccurate

### DISCUSSION THREAD

**Discussion by imxw** [ZuodaoTech/everyone-can-use-english#877] [Q&A]
It should be /hæz/ but what is obtained is /həz/ . 
![image](https://github.com/user-attachments/assets/cba98fd2-cf62-478e-8493-3e6e882794f7)

**Comment by bluechallenge** [comment]
这个应该没问题，has 可以弱读成 /həz/，此外，在日常的口语里，/h/ 读得很轻，所以 has 有时就直接弱读成 /əz/.

https://dictionary.cambridge.org/dictionary/english/has
